components/org.wso2.carbon.identity.oauth.par/src/main/java/org/wso2/carbon/identity/oauth/par/internal/ParServiceComponent.java

```diff
@@ -41,6 +42,8 @@ protected void activate(ComponentContext context) {
         try {
             context.getBundleContext().registerService(ParAuthService.class.getName(),
                     new ParAuthServiceImpl(), null);
```

Comment:
Add this to a data holder and use it instead of the one in ParUtil, refer https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2127/files#r1288034584